tasks/cea-install.js

```diff
@@ -48,35 +51,82 @@ function install() {
   });
 }
 
+// Create new package.json
+const npmInit = async () => {
+  const templatePkg = require(path.join(__dirname, '..', 'package.json'));
+  const appPkg = {
+    name: TARGET_DIR,
+    version: '0.1.0',
+    private: true
+  };
+
+  appPkg.scripts = templatePkg.scripts;
+  appPkg.devDependencies = templatePkg.devDependencies;
+  appPkg.dependencies = templatePkg.dependencies;
+  appPkg.eslintConfig = templatePkg.eslintConfig;
```

Comment:
could you clarify what `eslintConfig` is here?  I think just copying the dot file is good for our needs here?

Reply:
That was from a previous PR where I used a custom eslint object to point to the ./config directory